students/grzegorz_bajorski/lesson_03_functions/input_validation.py

```diff
@@ -0,0 +1,20 @@
+def collatz(number):
+    if number % 2 == 0:
+        return number // 2
+    else:
+        return 3 * number + 1
+
+print('Enter number')
+number = 0
+
+while 1:
+    try:
+        number = int(input())
+        if collatz(number) != 1:
+            print(collatz(number))
+        else:
+            break
+    except:
```

Comment:
Too broad exception clauses. When you catching exceptions you should mention specific exceptions whenever it is possible. Here as it is suggested in the task use ValueError.